synapse/config/sso.py

```diff
@@ -36,13 +36,27 @@ class SsoAttributeRequirement:
     """Object describing a single requirement for SSO attributes."""
 
     attribute: str
-    # If a value is not given, than the attribute must simply exist.
-    value: Optional[str]
+    # If a value or values are not given, than the attribute must simply exist.
+    value: Optional[str] = None
+    values: Optional[List[str]] = []
```

Comment:
rather than having both of these, could we just have a `values: List[str]`, and convert `value` into a single-item list?